src/igvfd/schemas/changelogs/measurement_set.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`measurement_set.json`*
 
+### Schema version 37
+
+* Adjust `preferred_assay_title` enum list to replace `Parse Perturb-seq` with `CC-Perturb-seq`.
```

Comment:
Minor typo in the property name, `preferred_assay_titles` since it's plural now

Reply:
fixed